docs/GettingStarted.md

```diff
@@ -1,80 +1,48 @@
-
-* For more detailed documentation please refer to the [User Guide](UserGuide.md)
-
-* If you are interested in extending / modifying the generator itself please refer to the [Developer Guide](DeveloperGuide.md)
-
 ## Downloading the JAR file
 
-The generator has been written in Java, allowing it to work on Microsoft Windows, Apple Mac and Linux. You will need Java v1.8 installed to run the generator (you can run `java -version` to check whether you meet this requirement), it can be [downloaded here](https://www.java.com/en/download/manual.jsp).
-
-The generator is distributed as a JAR file, with the latest release always available from the [GitHub releases page](https://github.com/finos/datahelix/releases/). The project is currently in beta and under active development. You can expect breaking changes in future releases, and new features too!
+The datahelix is distributed as a JAR file, with the latest release always available from the [GitHub releases page](https://github.com/finos/datahelix/releases/). You will need Java v1.8 installed to run the datahelix (you can run `java -version` to check whether you meet this requirement), it can be [downloaded here](https://www.java.com/en/download/manual.jsp).
 
-You are also welcome to download the source code and build the generator yourself. To do so, follow the instructions in the [Developer Guide](DeveloperGuide.md).
+You are also welcome to download the source code and build the generator yourself. To do so, follow the instructions in the [Developer Guide](DeveloperGuide.md#Building).
 
-Your feedback on the beta would be greatly appreciated. If you have any issues, feature requests, or ideas, please share them via the [GitHub issues page](https://github.com/finos/datahelix/issues).
+Datahelix is under active development so expect new features and bug fixes. Please feel free to share any issues, feature requests, or ideas via the [GitHub issues page](https://github.com/finos/datahelix/issues).
 
 ## Creating your first profile
 
+We are going to work through creating a profile to generate random personal data about some test users.
+
 Profiles are JSON files that describe the data you want to generate. They are composed of:
 
 -   **fields** - an array of uniquely named fields (or columns).
--   **rules** - an array of constraints, with an optional description.
--   **constraints** - restrict the types and ranges of data permitted for the given column.
+-   **constraints** - an array of restrictions on the types and ranges of data permitted for the given column.
 
-We'll start by generating data for a trivial schema. Using your favourite text editor, create the following JSON profile and save it as `profile.json`:
+We'll start by creating a simple profile containing a single field `username` with no constraints. Using  your favourite text editor, create the following JSON profile and save it as `profile.json`:
 
 ```json
 {
     "fields": [{ "name": "firstName", "type": "string" }],
```

Comment:
I'm still not sure that `firstName` is a good idea to use in this guide.  I like the approach of starting like this, starting with no constraints at all, showing that it can output anything at all and then introducing constraints to make it resemble your data more.  However the fact that it just ends with something that effectively says "actually we have a `firstname` data type, you should really just want to use that" makes it all a bit anticlimatic.

Maybe `username` would be a better idea for the example, because it is sufficiently similar to `firstName` but isn't duplicated by a type.

Reply:
(and you could then introduce `firstName` and `surname` as further fields)